### PR TITLE
feat: added todo web frontend

### PR DESCRIPTION
**Pull request by Rudra-Patel-code**
created a frontend part of todo app, using vite and typescript in react and tailwind css

### CODE REVIEW

**Comment by Rudra-Patel-code** [comment]
Do let me know if there are changes to be done

File: examples/todo-app/web/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# React + TypeScript + Vite
```
Comment (wajeshubham) [inline examples/todo-app/web/README.md:1]: Update the `README.md` with correct instruction on how to do the setup and installation including env file configuration

File: examples/todo-app/web/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
```
Comment (wajeshubham) [inline examples/todo-app/web/index.html:7]: Update the title based on the application

**Review by wajeshubham — CHANGES_REQUESTED** [review]
The UI is not up to the mark. It would be great if you could take a look at the following UI for the todo app. Please redesign the todo app based on the provided UI.

Todo app UI: https://devuiv2.vercel.app/templates/todo (Take a look at the other application screens for input and button components.)

**Pull request closed without merging**